lib/decanter/version.rb

```diff
@@ -1,3 +1,3 @@
 module Decanter
-  VERSION = '0.9.2'
+  VERSION = '0.9.3'
```

Comment:
@francirp Sorry for the delay. Do we feel like this a good time to bump this up to an official `1.0.0` release?